doc/user.md

```diff
@@ -0,0 +1,78 @@
+# User Guide
+
+## Isaac System
```

Comment:
Can we add instructions on how to run the example [Franka world](https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/tutorial_ros2_moveit.html)?

What command should the user run [here](https://docs.omniverse.nvidia.com/app_isaacsim/app_isaacsim/tutorial_ros2_moveit.html#running-moveit-2)? Does this depend on the `isaac_moveit` package? I though I remember you saying you can use the `moveit_resources` one.